app/parse.py

```diff
@@ -15,8 +22,94 @@ class Product:
     num_of_reviews: int
 
 
+def get_driver() -> WebDriver:
+    options = Options()
+    options.headless = True
+    service = Service(executable_path="C:/chromedriver-win64/chromedriver.exe")
```

Comment:
please don't use your absolute path, script should be universal and work on all OS